site/content/3.11/arangograph/deployments/private-endpoints.md

```diff
@@ -100,9 +107,15 @@ please contact support via **Request help** in the help menu.
    changed anymore once a connection has been established.
    Proceed by clicking **Next**.
    ![ArangoGraph Deployment Private Endpoint Setup 2](../../../images/arangograph-deployment-private-endpoint-setup2.png)
-6. Enter one or more Alternate DNS names. This step is optional.
-   Continue with or without Alternate DNS names entered by clicking **Next**.
-   They can be changed later.
+6. Configure custom DNS names. This step is optional and disabled by default,
+   you can also add or change them later from the summary view.
+   Click **Next** to continue.
+   {{< info >}}
+   By default, your private endpoint is available to all VPCs that connect to it
+   at `https://<endpoint_id>-pe.arangodb.cloud` with the well-known certificate.
+   If the custom DNS is enabled, you will be responsible for the DNS of your
+   private endpoints. 
+   {{< /info >}}
```

Comment:
this feature isn't available on Azure, on AWS (can be changed at runtime if a customer like) and Google (where it's an immutable setting after creation of the PE)